benchmark/string_decoder/string-decoder.js

```diff
@@ -9,7 +9,7 @@ var bench = common.createBenchmark(main, {
   n: [25e4]
 });
 
-var UTF_ALPHA = 'Bl�b�rsyltet�y';
+var UTF_ALPHA = 'Blåbærsyltetøy';
```

Comment:
Looks like file encoding was changed from ISO 8859-15 to UTF-8.
Is that intended?